spec/lib/submission_builder/ty2024/states/id/documents/id40_spec.rb

```diff
@@ -0,0 +1,14 @@
+require 'rails_helper'
+
+describe SubmissionBuilder::Ty2024::States::Id::Documents::Id40, required_schema: "id" do
+  describe ".document" do
+    let(:intake) { create(:state_file_id_intake, filing_status: "single") }
+    let(:submission) { create(:efile_submission, data_source: intake) }
+    let(:build_response) { described_class.build(submission, validate: false) }
```

Comment:
b/c `validate` is false, wouldn't the response always be a valid xml? Is this intentional b/c the fields haven't been filled out yet?

Reply:
true! it's mostly empty so at this point will never be valid. actually i just copied the md equivalent for this and didn't realize that the test itself expects it to be "valid" haha—will modify it to make a bit more sense